driver/src/main/java/org/neo4j/driver/internal/cluster/Rediscovery.java

```diff
@@ -232,9 +232,7 @@ else if ( composition != null )
             }
             else
             {
-                ClusterComposition cluster = response.clusterComposition();
-                logger.info( "Got cluster composition %s", cluster );
-                return cluster;
+                return response.clusterComposition();
```

Comment:
Consider logging records received from routing procedure on debug level

Reply:
If debug level is enabled, then the routing table will be printed out in the record message coming from getRoutingTable procedure. Therefore there might be no need to log again in debug level.

Instead we might want a logger at `INFO` level when there is a new routing table if we only have INFO level on.